### DISCUSSION TITLE
PatchTool 6.18 8/9

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#203] [Announcements]
1. 更新文本至中國服6.1進度，並手動檢查大部分簡繁轉換內容。
2. 修正沙斯塔夏任務。感謝 #197 提供的資訊。
3. 修正舊文本與簡繁轉換。
4. 修正指令碼錯誤。

[Wiki](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/5.-CSV) is updated.

檔案：
- FFXIVChnTextPatch-0809.rar：本體。一般而言，下載這個即可。JRE縮小過。

選用：
- rawexd-zhcn-0809.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- [第三方4K字體包](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.53/font-4k.rar)：部分解析度設定下會出現字體錯位等情形，請自行照需求使用。解壓後改名為font放到resource資料夾下（取代原資料夾）。
- [FFXIV-jar-0211.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.56/FFXIV-jar-0211.rar)：如果你有遇到JNI error，請參考 #90 放到與exe同位置使用。
- [backup-0706.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.65/backup-0706.rar)：乾淨的0a0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v2.66'>PatchTool 6.18 8/9</a>.</em>

**Comment by KppKerry** [comment]
辛苦了~感謝大佬

**Comment by tonyjediex** [comment]
有您真好

**Comment by ClareTW** [comment]
感謝

**Comment by okami2223** [comment]
不好意思，8/23今日更新後再打倒光之戰士阿爾伯特之後出現了錯誤... 還原後即可正常過劇情
![image](https://user-images.githubusercontent.com/44223291/186198273-b5546dd2-80d4-4b23-8cf1-368fcdd4b8e2.png)

**Reply by GpointChen** [reply]
已更新

**Reply by okami2223** [reply]
> 已更新

謝謝! 已測試可順利看劇情了，辛苦你了!
